aclk/agent_cloud_link.c

```diff
@@ -763,7 +767,7 @@ int aclk_init(ACLK_INIT_ACTION action)
     aclk_recv_maximum = config_get_number(CONFIG_SECTION_ACLK, "agent cloud link receive maximum", 20);
 
     aclk_hostname = config_get(CONFIG_SECTION_ACLK, "agent cloud link hostname", "localhost");
-    aclk_port = config_get_number(CONFIG_SECTION_ACLK, "agent cloud link port", 1883);
+    aclk_port = config_get_number(CONFIG_SECTION_ACLK, "agent cloud link port", 9002);
 
     info("Maximum parallel outgoing messages %d", aclk_send_maximum);
```

Comment:
A suggestion, add some common identifier for messages coming from ACLK, for example, when we are talking about stream we always have `[STREAM] ` in the messages.

Reply:
@thiago you mean prefix every ACLK log msg with "[ACLK]" string to be able to easily filter it in log?

Reply:
Yes, this will help us to know the Netdata's area that is reporting the problem. :)

Reply:
This is running on a thread that will be reporting the thread name. For example:
```
2020-02-06 20:23:15: netdata INFO  : ACLKQ : Waiting for agent to get to stable state
2020-02-06 20:23:23: netdata INFO  : ACLKQ : Waiting for agent to get to stable state
2020-02-06 20:23:31: netdata INFO  : ACLKQ : Waiting for agent to get to stable state
2020-02-06 20:23:39: netdata INFO  : ACLKQ : AGENT is stable
```